tests/expressions/tests.py

```diff
@@ -2045,3 +2048,124 @@ def test_hash(self):
             hash(OrderBy(F('field'), nulls_last=True)),
             hash(OrderBy(F('field'), nulls_last=False)),
         )
+
+
+class ComparisonExpressionTests(TestCase):
```

Comment:
`tests/expressions` already contains a mix of tests. I would move them to the `tests/lookup/tests.py`